openvpn/start.sh

```diff
@@ -219,8 +219,8 @@ else
 fi
 
 if [[ -f /run/secrets/rpc_creds ]]; then
-  export TRANSMISSION_RPC_USERNAME=$(head -1 /run/secrets/rpc_creds)
-  export TRANSMISSION_RPC_PASSWORD=$(tail -1 /run/secrets/rpc_creds)
+  export TRANSMISSION_RPC_USERNAME=$(sed -n '1p' /run/secrets/rpc_creds)
+  export TRANSMISSION_RPC_PASSWORD=$(sed -n '2p' /run/secrets/rpc_creds)
```

Comment:
i set up this image recently and I was a bit surprised that these are not set individually with the possibility of using `_FILE` declarations.